doc/developer/index.adoc

```diff
@@ -36,6 +36,24 @@ release while the _develop_ version uses the next expected major/minor
 release number, such as 2.1-SNAPSHOT and 2.2-SNAPSHOT respectively
 (after a 2.1.x release, but before any 2.2.x releases are made public).
 
+== Subprojects
+
+RDF4J is split between five github projects. Those are:
+
+* https://github.com/eclipse/rdf4j
+* https://github.com/eclipse/rdf4j-storage
+* https://github.com/eclipse/rdf4j-tools
+* https://github.com/eclipse/rdf4j-testsuite
+* https://github.com/eclipse/rdf4j-doc
+
+Each of these github projects has their own set of Issues, Pull Requests, Releases, artifacts, milestones, and branches. Issues should be created in the coresponding subproject that most closely fits.
```

Comment:
Are you sure this is a good idea? People are sloppy, you're bound to get issues in the "wrong" project. Might be better to just keep a single issue tracker and have everything in one place. 

Note that you can still easily reference an issue in another project via commit messages or github comments, see https://help.github.com/articles/autolinked-references-and-urls/

Reply:
I intend on keeping existing issues where they are and I'll try not to be to strict about bug reporting as we can uses these cross-github-links anyway. However, I think contributors would have a harder time adjusting to cross-github-links correctly than using multiple issues lists. I think it is important to be consistent and I've noticed other large projects (like bootstrap, django, and nodejs) that have multiple github projects also use multiple issues lists and I think that is the norm on github.